hw/ip/otbn/code-snippets/LICENCE.dcrypto

```diff
@@ -0,0 +1,30 @@
+Some code in this directory originally came from the Chromium project.
```

Comment:
Slight better practice to leave the license fil 100% unmodified

Reply:
Done.